### DISCUSSION TITLE
Contact Libre Arts?

### DISCUSSION THREAD

**Discussion by REMC19** [Aries85/LightZone#280] [General]
Masahiro, would you have any objections to Alexandre Prokoudine (of librearts.org) being approached regarding LightZone - as an attempt to raise the profile of the software and work to gain it the wider audience it (and your efforts) deserve.
As Librearts.org's about states the site itself is "an online magazine for creative professionals using free applications for digital painting, graphic and web design, desktop publishing, photography, and CAD. The project focuses on news, tutorials and articles to provide you with the most up to date information about evolution of these applications and best practices."

**Comment by ktgw0316** [comment]
@REMC19 Thank you for the offer. Sounds great to me.

**Reply by REMC19** [reply]
Hi Masahiro.
Just letting you know I have sent a preliminary email to Alexandre at LibreArt.org to enquire if he has any interest in producing an article regarding LightZone.
I have informed him that if he does I will attempt to gather what information I can for him
